application/src/main/kotlin/org/gxf/crestdevicesimulator/configuration/SimulatorProperties.kt

```diff
@@ -20,4 +20,5 @@ class SimulatorProperties(
         val successMessage: Resource,
         val failureMessage: Resource,
         val produceValidCbor: Boolean,
+        val cipherSuite: String
```

Comment:
I believe this should also work:
```suggestion
        val cipherSuite: List<CipherSuite>
```